changelogs/current.yaml

```diff
@@ -246,6 +246,10 @@ bug_fixes:
 - area: datadog
   change: |
     Bumped the version of datadog to resolve a crashing bug in earlier versions of the library.
+- area: ext_authz
+  change: |
+    Fixed fail-open behaviour of the failure_mode_allow config option when a grpc external authz server is used.
```

Comment:
Please add a link to the `failure_mode_allow` field in the docs

Reply:
Looks like the https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto page doesn't have anchor links, so I added just a link to the page where this param can be found